extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -71,10 +99,84 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
         // Ignore messages from other iframes.
         return;
       }
-      // Player wants to close because the user interracted on its close button
-      if (getData(e) === 'cnx_close') {
-        this.destroyPlayerFrame_();
-        this.attemptCollapse();
+      const dataString = getData(e);
+      const dataJSON = tryParseJson(dataString) || dict();
+
+      if (dataJSON['event'] === 'command') {
+        switch (dataJSON['func']) {
+          // Player wants to close because the user interracted on its close button
+          case 'cnxClose': {
+            this.destroyPlayerFrame_();
+            this.attemptCollapse();
+            break;
+          }
+          // Player rendered
+          case 'cnxPlayerRendered': {
+            this.playerReadyResolver_(this.iframe_);
+            break;
+          }
+        }
+      }
+    });
+  }
+
+  /**
+   * Binds to amp-consent
+   * @private
+   */
+  bindToAmpConsent_() {
```

Comment:
@zhouyx might have more input on the consent integration but looks good to me

Reply:
Thanks @wassgha. How it can have more input on the consent integration?